server/src/models/diesel.rs

```diff
@@ -17,7 +17,7 @@ pub(crate) enum ProposalCategory {
     Core,
     Networking,
     Interface,
-    ERC,
+    Erc,
```

Comment:
I suspect that this Enum (ProposalCategory) is going to be written verbatim to the UI. In that case, "ERC" should indeed match the documentation elsewhere (Mina's MIP repo). In that case, perhaps the pragma that tells the linter to allow ERC (and not require "Erc") should indeed be used, here, preferably locally and not merely on this entire file. Alternately, you could explicity specify how to "show" or "serialize" these things, such that "Erc" serializes to "ERC". In any case, consistency with the MIP documentation is important, and takes precedence over the Rust naming conventions that the linter is trying to ensure.

Reply:
@robinbb I reverted and and the macro https://github.com/Granola-Team/mina-on-chain-voting/pull/242/commits/97cbc0c548000d343365f568214055da19b5957e and https://github.com/Granola-Team/mina-on-chain-voting/pull/242/commits/baa8705b9eae632ba0564b89c56e2a1b7791c4dc